### ISSUE TITLE
Accessibility: Prioritize accessibility tickets in the backlog

### ISSUE THREAD

**Issue by Katherine-Osos** [cisagov/manage.get.gov#2933]
### Issue description

We have a variety of accessibility tickets in the backlog for design and dev. We need to go through all tickets and assign a priority. Can use the "priority" field and/or another way to show priority for the team.

Tickets related to the 508 compliance should be top priority regardless of accessibility impact. All other tickets can be assessed based on impact.

### Acceptance criteria

- [ ] Establish plan for how priority will be assigned
  - [ ] Share and confirm plan with product team
- [ ] Review all accessibility tickets and assign priority; consider design and dev tickets.
- [ ] As needed, document process/considerations on how priority should be assigned in the future
